both/_collections.js

```diff
@@ -286,6 +286,18 @@ Registrations.attachSchema({
   },
   uniqueName: {
     type: String,
+    unique: true,
```

Comment:
Is this correct? Won't this prevent someone from registering for 2 competitions?